packages/ui/src/ui/pages/system/Masters/ChangeMaintenanceButton.tsx

```diff
@@ -1,55 +1,52 @@
-import React, {FC, useCallback, useState} from 'react';
+import React, {FC, useState} from 'react';
 import {Button, Icon} from '@gravity-ui/uikit';
-import {changeMasterMaintenance} from '../../../store/actions/system/masters';
-import {changeSchedulerMaintenance} from '../../../store/actions/system/schedulers';
-import {useDispatch} from 'react-redux';
 import PencilIcon from '@gravity-ui/icons/svgs/pencil.svg';
 import {FormApi, YTDFDialog, makeErrorFields} from '../../../components/Dialog';
-import hammer from '../../../common/hammer';
 import {useToggle} from 'react-use';
 
 type Props = {
     className?: string;
-    address: string;
+    path: string | null;
+    title?: string;
```

Comment:
The title might be always generated from `path`:
```ts
const address = path?.split('/').pop();
const title = `Edit ${address}`;
```
Do we really need the property?

Reply:
The header is formed differently for Scheduler and Masters. That's why I made it even higher